blackbox/docs/sql/statements/set-license.rst

```diff
@@ -0,0 +1,60 @@
+.. highlight:: psql
+.. _ref-set-license:
+
+===============
+``SET LICENSE``
+===============
+
+Sets the license Key.
+
+.. rubric:: Table of Contents
+
+.. contents::
+    :local:
+
+Synopsis
+========
+
+::
+
+    SET LICENSE license_key
+
+Description
+===========
+
+``SET LICENSE`` registers a CrateDB license key to the cluster.
+If the license registration is successful, CrateDB will continue to operate
+in `Enterprise Edition`_ until the license expiry date.
+
+.. NOTE::
+
+    Should the license registration fail, the cluster will continue to
+    operate in the existing mode e.g. ``Enterprise`` as long as
+    the current license is still valid (i.e. not expired).
+
+    When the existing licence expires, the cluster will run in a *degraded mode*,
+    allowing only read only system information operations and the
+    ``SET LICENSE`` statement.
+
+.. SEEALSO::
+    - :ref:`enterprise_features`
+
+Parameters
+==========
+
+License Key
+-----------
+
+The ``license_key`` parameter is the encrypted representation of a CrateDB license.
```

Comment:
I'd not go into the encryption details here. I don't think that provides any useful information to the user (and it's fairly inaccurate as well as it's the encoded license information - which happens to contain an encrypted bit)

I'd keep this brief and maybe talk about what the license means for the product not what it actually is.

Reply:
I am OK to keep it brief but talking about what license means for the product at this place (parameters) also seems a bit odd. 

I'll move this info in the sys.cluster session as @mfussenegger suggested